CHANGELOG.md

```diff
@@ -43,6 +43,17 @@
   - Changed `.a-avatar--medium` to `.a-avatar--l`
   - Changed `.a-avatar--large` to `.a-avatar--xl`
 
+#### Accordion
+
+##### Added
+- Added round variant for every avatar
+
+##### Changed
+- Implemented new design changes
+- [BREAKING] Replaced SASS variables by CSS properties
+- [BREAKING] Changed various CSS classes:
+  - Removed `.has-margins`
```

Comment:
Indentation ;-)

Reply:
Die indentation is bewust gedaan. Ik wil bewust een inspringing om . 🙂 

Bij de andere componenten in de changelog doe ik dat ook. Bijvoorbeeld bij de avatar:

```
- [BREAKING] Changed various CSS classes:
  - Changed `.a-avatar--small` to `.a-avatar--s`
  - Changed `.a-avatar--medium` to `.a-avatar--l`
  - Changed `.a-avatar--large` to `.a-avatar--xl
`
```

<img width="490" alt="Screenshot 2021-12-02 at 15 38 01" src="https://user-images.githubusercontent.com/4232875/144442998-4c1645b4-cf59-44d2-95f6-d7a798678405.png">